src/test/java/stepsDesign/GCSBasicDemo.java

```diff
@@ -117,21 +119,23 @@ public void saveAndDeployPipeline() throws InterruptedException {
     }
 
     @Then("Run the Pipeline in Runtime")
-    public void runThePipelineInRuntime() {
+    public void runThePipelineInRuntime() throws InterruptedException {
         CdfPipelineRunAction.runClick();
     }
 
     @Then("Wait till pipeline is in running state")
     public void waitTillPipelineIsInRunningState() throws InterruptedException {
         Boolean bool= true;
         WebDriverWait wait = new WebDriverWait(SeleniumDriver.getDriver(), 1000000);
-        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@data-cy='Succeeded']")));
+        wait.until(ExpectedConditions.or(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@data-cy='Succeeded']")),
+                ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@data-cy='Failed']"))));
     }
 
     @Then("Verify the pipeline status is {string}")
     public void verifyThePipelineStatusIs(String status) {
-
-        Assert.assertTrue( SeleniumDriver.getDriver().findElement(By.xpath("//*[@data-cy='"+status+"']")).isDisplayed());
+        webelement=false;
```

Comment:
This can be a local variable as it's not used anywhere else.

Reply:
Made it local variable